pkg/trace/api/debugger.go

```diff
@@ -0,0 +1,91 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package api
+
+import (
+	"fmt"
+	stdlog "log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/trace/info"
+	"github.com/DataDog/datadog-agent/pkg/trace/logutil"
+	"github.com/DataDog/datadog-agent/pkg/trace/metrics"
+	"github.com/DataDog/datadog-agent/pkg/util/fargate"
+)
+
+const (
+	// logsIntakeURLTemplate specifies the template for obtaining the intake URL along with the site.
+	logsIntakeURLTemplate = "https://http-intake.logs.%s/v1/input"
+	// logsIntakeDefaultURL specifies the default intake API URL.
+	logsIntakeDefaultURL = "https://http-intake.logs.datadoghq.com/v1/input"
+)
+
+// debuggerProxyHandler returns an http.Handler proxying requests to the logs intake. If the logs intake url cannot be
+// parsed, the returned handler will always return http.StatusInternalServerError with a clarifying message.
+func (r *HTTPReceiver) debuggerProxyHandler() http.Handler {
+	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, info.Version)
+	if orch := r.conf.FargateOrchestrator; orch != fargate.Unknown {
+		tag := fmt.Sprintf("orchestrator:fargate_%s", strings.ToLower(string(orch)))
+		tags = tags + "," + tag
+	}
+	intake := logsIntakeDefaultURL
```

Comment:
I think we can drop `logsIntakeDefaultURL` and just default this value to:
```suggestion
	intake := fmt.Sprintf(logsIntakeURLTemplate, config.DefaultSite)
```
What do you think?